arclightning_frontend/games.html

```diff
@@ -27,17 +40,24 @@
 		<h1>Arc Lightning</h1>
 	   </div>
             <div class="row" id="games">
-                <div class="col-sm-2" style="border: 1px solid black; height:42vh; padding: 10px; margin-bottom:15px; margin-left: 20px; margin-right:20px;" v-for="item in items">
+                <div class="col-sm-2 unselectedClass" v-for="(item,index) in items" v-bind:class="{ selectedClass: index == selectedIndex }" >
+	
 		<img style="display:block;position:relative;height:90%; width:90%; margin-left: auto; margin-right:auto; "src v-bind:src= "item.thumbnail_path"  />
-                <div style="text-align: center; border-top: 1px solid black; font-family: 'Do Hyeon', sans-serif; font-size: 24px;"> {{ item.name }} </div>	
+                <div style="text-align: center; border-top: 1px solid black; font-family: 'Do Hyeon', sans-serif; font-size: 24px;"> {{ item.name }}</div>	
+                
 		</div>
             </div>
         </div>
     </body>
     <script type="text/javascript">
         var app = new Vue({
             el: '#games',
+	    created(){
+		this.onKeyDown = this.onKeyDown.bind(this);
+		document.addEventListener('keydown', this.onKeyDown);
+	    },
             data: {
+		selectedIndex: 0,
                 items: [
```

Comment:
We'll need to restructure this as a dictionary

Something like
```js
{
  'smash':  {name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png'},
  'touhou':  {name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png'},
}
```
where a unique key id identifies each game and the value represents game info. Also remove exe_path, since the client doesn't need that info.